src/test/three-components/SmoothControls-spec.ts

```diff
@@ -363,16 +366,16 @@ suite('SmoothControls', () => {
               USER_INTERACTION_CHANGE_SOURCE,
               USER_INTERACTION_CHANGE_SOURCE,
             ];
-            let changeSource = [];
+            let changeSource: Array<string> = [];
 
             controls.addEventListener('change', ({source}) => {
               changeSource.push(source);
             });
 
             dispatchSyntheticEvent(element, 'keydown', {keyCode: KeyCode.UP});
-            controls.update(performance.now, ONE_FRAME_DELTA);
-            controls.update(performance.now, ONE_FRAME_DELTA);
-            controls.update(performance.now, ONE_FRAME_DELTA);
+            controls.update(performance.now(), ONE_FRAME_DELTA);
```

Comment:
Considering the first argument is unused, might we just pass 0?

Reply:
We know the method's implementation, so technically we might. We go through the motions here and pretend not to know them, because the contract is the method signature (not its implementation).